app/cocina_builders/related_links_cocina_builder.rb

```diff
@@ -25,16 +25,16 @@ def call
   attr_reader :related_links
 
   def related_link_params_for(related_link:)
-    {
-      access: {
-        url: [
-          { value: related_link['url'] }
-        ]
-      }
-    }.tap do |related_link_hash|
-      next if (related_link_text = related_link['text']).blank?
-
-      related_link_hash[:title] = [{ value: related_link_text }]
+    {}.tap do |related_link_hash|
+      related_link_text = related_link['text']
```

Comment:
Should this be:

```suggestion
      related_link_text = related_link_hash['text']
```

❓ And if so, should the test suite be catching that?

Reply:
Nevermind, I think I grok what's going on here better now. 😄